pages/cockpit/overview/introduction.adoc

```diff
@@ -7,37 +7,32 @@
 
 == Overview
 
-Gravitee Cockpit is a centralized, multi-environment tool for managing all your Gravitee API Management and Access Management installations in one handy, interactive dashboard.
+Gravitee Cockpit is a centralized, multi-environment tool for managing all your Gravitee API Management and Access Management installations in a single place.
 
-.Cockpit dashboard
 image::cockpit/cockpit-dashboard.png[]
 
 [[organizational-hierarchy]]
 == Cockpit hierarchy
 
 Cockpit is based on a hierarchy of three entity types:
 
-* The top level is an _account_ (corresponding to a company)
-* The next level is an _organization_ (corresponding to a logical part of the company in the particular company context, such as a region or business unit)
-* The lowest level is an _environment_ (corresponding to an environment in an IT infrastructure, such as a development or production environment)
+* *Account*: at the top level, your company. There is only one account.
```

Comment:
Actually one user account can be linked to multiple `Accounts`
In that case you will have a selector in the breadcrumb like this

![screenshot-localhost_4004-2021 11 24-16_35_24](https://user-images.githubusercontent.com/1655950/143268381-07b858c9-234e-4d55-b704-442645b62638.png)